### ISSUE TITLE
fortios_system_dhcp_server connection error

### ISSUE THREAD

**Issue by raptorr** [fortinet-ansible-dev/ansible-galaxy-fortios-collection#129]
I try to add configuration to dhcp with task:
```yaml
- hosts: '{{ target | d("undefined")  }}'
  connection: httpapi
  collections:
    - fortinet.fortios
  vars:
    ansible_network_os: fortios
    ansible_httpapi_use_ssl: yes
    ansible_httpapi_validate_certs: no
    ansible_httpapi_port: 443
    dhcp_opt_id: 100
    vdom: "root"

  tasks:

  - name: Set dhcp options for int
    fortinet.fortios.fortios_system_dhcp_server:
      state: present
      system_dhcp_server:
        options:
         -
            id: '{{ dhcp_opt_id }}'
            code: "43"
            type: "hex"
            value: "010400000002"
        interface: "INT"
        id: "100"
```
But it ends with error:
```bash
The full traceback is:
Traceback (most recent call last):
  File "<stdin>", line 102, in <module>
  File "<stdin>", line 94, in _ansiballz_main
  File "<stdin>", line 40, in invoke_module
  File "/usr/lib64/python3.9/runpy.py", line 210, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib64/python3.9/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib64/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_dhcp_server.py", line 4219, in <module>
  File "/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_dhcp_server.py", line 4196, in main
  File "/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/module_utils/fortios/fortios.py", line 248, in check_schema_versioning
  File "/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible/module_utils/connection.py", line 185, in __rpc__
ansible.module_utils.connection.ConnectionError: Method not found
fatal: [fortigate]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"<stdin>\", line 102, in <module>\n  File \"<stdin>\", line 94, in _ansiballz_main\n  File \"<stdin>\", line 40, in invoke_module\n  File \"/usr/lib64/python3.9/runpy.py\", line 210, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib64/python3.9/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/lib64/python3.9/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_dhcp_server.py\", line 4219, in <module>\n  File \"/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/modules/fortios_system_dhcp_server.py\", line 4196, in main\n  File \"/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible_collections/fortinet/fortios/plugins/module_utils/fortios/fortios.py\", line 248, in check_schema_versioning\n  File \"/tmp/ansible_fortinet.fortios.fortios_system_dhcp_server_payload_c1hsq3z7/ansible_fortinet.fortios.fortios_system_dhcp_server_payload.zip/ansible/module_utils/connection.py\", line 185, in __rpc__\nansible.module_utils.connection.ConnectionError: Method not found\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
```
I try run it on FortiGate-100D v6.2.5

**Comment by raptorr** [comment]
Issue occurs on ansible 2.9.25, on 2.11.2 works correctly

**Comment by JieX19** [comment]
Hi @raptorr  I'm sorry for the late reply. Yep, the bug has been fixed since ansible 2.11. Thank you for your feedback. Free free to open a new issue if you have any questions.
